bin/upload-data.py

```diff
@@ -71,6 +71,11 @@ def main():
         print("Folder is relative, using", folder)
     if args.common is not None:
         copy_everywhere(config, args.common, folder, copyOptions)
+    if args.files:
+        copy_files(config, folder, args.files, copyOptions)
+    else:
+        print("No files to upload to the machines provided in a Hillview configuration")
+        exit(1)
```

Comment:
There is no need to exit with 1 - sometimes you only want to use the args.common option

Reply:
@mbudiu-vmw Done.